test/extended/operators/OWNERS

```diff
@@ -0,0 +1,20 @@
+reviewers:
+  - smarterclayton
+  - bparees
+  - mfojtik
+  - gabemontero
+  - deads2k
+  - knobunc
+  - ironcladlou
+  - adambkaplan
+  - kevinrizza
+  - ecordell
+approvers:
+  - smarterclayton
+  - bparees
+  - mfojtik
+  - knobunc
+  - ironcladlou
+  - adambkaplan
+  - kevinrizza
+  - ecordell
```

Comment:
you should only list OLM people here.

it's inherited, so i'll still have permissions from the top level files.